include/curl/curl.h

```diff
@@ -1866,6 +1866,23 @@ enum CURL_NETRC_OPTION {
   CURL_NETRC_LAST
 };
 
+#define GET_CURL_SSLVERSION(x) (x & 0xffff)
+#define GET_CURL_SSLVERSION_MAX(x) (x & 0xffff0000)
+#define SET_CURL_SSLVERSION_MAX(x, val) \
+  CURL_SSLVERSION_MAX_##x = (val << 16)
+
+enum {
+  SET_CURL_SSLVERSION_MAX(NONE,    0),
+  SET_CURL_SSLVERSION_MAX(FIRST,   1),
+  SET_CURL_SSLVERSION_MAX(TLSv1_0, 1),
+  SET_CURL_SSLVERSION_MAX(TLSv1_1, 2),
+  SET_CURL_SSLVERSION_MAX(TLSv1_2, 3),
+  SET_CURL_SSLVERSION_MAX(TLSv1_3, 4),
+
+  SET_CURL_SSLVERSION_MAX(LAST,    5), /* never use, keep last */
+  SET_CURL_SSLVERSION_MAX(DEFAULT, 3)  /* = ..._OR_UP_TO_TLSv1_2 */
```

Comment:
Looks wrong to me.  If we changed the default in libcurl later on, all dependent software would need to be recompiled for the change to take effect.  You need to assign a separate value for ```CURL_SSLVERSION_MAX_DEFAULT``` so that a rebuilt of (dynamically linked) libcurl is sufficient to change the default.